testdata/p4_16_errors/table-entries-exact-ternary.p4

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
p4test runs all of them.
p4c-bm2-ss runs all the ones that end with bmv2.p4. And the output json is validated by the simulator. So you want to use it, because otherwise the json generation code is not exercised.